modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -220,6 +220,30 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 
 == Fixed security issues
 
+=== Fixed security issues in {product} 1.2.6
+
+This section lists fixed issues with {product} 1.2.6:
```

Comment:
Please replace hardcoded content with link to modules/release-notes/snip-fixed-security-issues-in-product-1.2.6.adoc like we did for 1.2.5 and 1.2.3, etc.

Reply:
Done! thanks.